### PR TITLE
feat(cards): Add Card API in 1st iteration

### PR DESCRIPTION
**Pull request by patrickvenetz**
Run migrations `yarn db:migrate:up` which generates mocked data.

Then import CSV files:

```
packages/cards/script/import-smartvote-data.js --data-url https://url-to-file
```

It will generate groups and cards, as well as users.

### CODE REVIEW

File: packages/cards/graphql/schema-types.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+module.exports = `
+
+type Card {
+  id: ID!
+  payload: JSON!
+  group: CardGroup!
+  user: User
```
Comment (tpreusse) [inline packages/cards/graphql/schema-types.js]:
```suggestion
  user: User!
```

I don't want to handle null users :)

File: packages/cards/graphql/schema-types.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  documents: DocumentConnection!
+  match: Boolean
+  totalMatches: Int!
```
Comment (tpreusse) [inline packages/cards/graphql/schema-types.js]:
```suggestion
  totalMatches: Int!
  statement: Comment
```

I want to query a preview of that comment

File: packages/cards/graphql/resolvers/_mutations/claimCard.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  const transaction = await pgdb.transactionBegin()
+  const now = new Date()
+
+  try {
+    const card = await transaction.public.cards.findOne({ id, userId: tokenUser.id })
+    if (!card) {
+      throw new Error('api/cards/claimCard/cardNotFound')
+    }
+
+    if (tokenUser.verified) {
```
Comment (patte) [inline packages/cards/graphql/resolvers/_mutations/claimCard.js]:
```suggestion
    if (tokenUser.verified && tokenUser.id !== user.id) {
```

File: packages/cards/graphql/resolvers/_mutations/claimCard.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        { id: user.id },
+        {
+          firstName: user._raw.firstName || tokenUser._raw.firstName,
+          lastName: user._raw.lastName || tokenUser._raw.lastName,
+          username: user._raw.username || tokenUser._raw.username,
+          roles: Array.from(
+            new Set([].concat(user._raw.roles).concat(tokenUser._raw.roles).filter(Boolean))
+          ),
+          hasPublicProfile: true,
+          portraitUrl: portraitUrl || tokenUser._raw.portraitUrl || user._raw.portraitUrl,
```
Comment (patte) [inline packages/cards/graphql/resolvers/_mutations/claimCard.js]: use https://github.com/orbiting/backends/blob/master/packages/assets/lib/Portrait.js#L66 to delete obsolete images from S3

File: packages/cards/graphql/resolvers/_mutations/updateCard.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+module.exports = async (
+  _,
+  { id, portrait, statement, payload },
+  context
+) => {
+  const { req, user, pgdb } = context
+  ensureSignedIn(req)
+
+  console.log({ id, portrait: portrait && portrait.length, statement, payload })
```
Comment (patte) [inline packages/cards/graphql/resolvers/_mutations/updateCard.js]:
```suggestion
```

File: packages/cards/graphql/resolvers/_mutations/updateCard.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+    if (portrait !== undefined) {
+      const portraitUrl = portrait ? await Portrait.upload(portrait) : null
+      await transaction.public.users.update(
+        { id: user.id },
+        {
+          portraitUrl: portraitUrl,
+          updatedAt: now
+        }
+      )
```
Comment (patte) [inline packages/cards/graphql/resolvers/_mutations/updateCard.js]: remove old `portraitUrl` from S3

File: packages/cards/graphql/resolvers/_mutations/updateCard.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+  try {
+    const card = await transaction.public.cards.findOne({ id, userId: user.id })
+    if (!card) {
+      throw new Error('api/cards/updateCard/cardNotFound')
+    }
+
+    if (portrait !== undefined) {
+      const portraitUrl = portrait ? await Portrait.upload(portrait) : null
+      await transaction.public.users.update(
```
Comment (patte) [inline packages/cards/graphql/resolvers/_mutations/updateCard.js]:
```suggestion
      await transaction.public.users.updateOne(
```

File: packages/cards/graphql/resolvers/_mutations/updateCard.js [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+          position: (position && String(position)) || ''
+        }))
+      }
+    }
+
+    await transaction.public.cards.update(
+      { id },
+      {
+        payload: updatedPayload,
+        userId: user.id,
```
Comment (patte) [inline packages/cards/graphql/resolvers/_mutations/updateCard.js]:
```suggestion
```

File: packages/cards/graphql/resolvers/_queries/cards.js [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+      Object.assign({}, defaults, args),
+      []
+    )
+  }
+
+  const cards = user
+    ? await loaders.Card.byUserId.load(user.id)
+    : await pgdb.public.cards.findAll()
+
+  console.log(cards)
```
Comment (patte) [inline packages/cards/graphql/resolvers/_queries/cards.js]:
```suggestion
```

**Pull request merged**